scripts/java/tools/dummytokens/build.gradle

```diff
@@ -15,6 +15,7 @@ dependencies {
 }
 
 jar {
+    duplicatesStrategy = DuplicatesStrategy.EXCLUDE
```

Comment:
Could you add comment why this is needed?

Reply:
The error was 
```
Gradle detected a problem with the following location: '/Users/anath1/IdeaProjects/openhouse_oss_07262024/openhouse/build/configs/libs/configs.jar'. Reason: Task
':scripts:java:tools:dummytokens:jar' uses this output of task ':cluster:configs:jar' without declaring an explicit or implicit dependency. This can lead to incorrect results being produced, depending on what order the tasks are executed.

Based on the [wiki](https://docs.gradle.org/7.6.2/userguide/validation_problems.html#implicit_dependency) provided the duplicate strategy this task depends on the output of another task and copying files to same destination.